pandas/core/generic.py

```diff
@@ -2930,25 +2931,61 @@ def blocks(self):
         """Internal property, property synonym for as_blocks()"""
         return self.as_blocks()
 
-    def astype(self, dtype, copy=True, raise_on_error=True, **kwargs):
+    def astype(self, dtype, copy=True, inplace=False, raise_on_error=True,
+               **kwargs):
         """
         Cast object to input numpy.dtype
-        Return a copy when copy = True (be really careful with this!)
 
         Parameters
         ----------
-        dtype : numpy.dtype or Python type
+        dtype : numpy.dtype or Python type (to cast entire DataFrame to the
+            same type). Alternatively, {col: dtype, ...}, where col is a column
+            label and dtype is a numpy.dtype or Python type (to cast one or
```

Comment:
@jreback `PandasObject` is not really meaningful towards users. But a more general "pandas object" (no code quoting) is maybe a possibility?

Further, @StephenKappel, can you format this according to the numpydoc standard? (https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt#sections):

```
param : type
    Description of parameter.
```